wimaging/wimaging_script_winpe_setup.erb

```diff
@@ -32,12 +36,12 @@ C:\Windows\System32\bcdboot C:\Windows /l en-US
  
 echo Downloading the drivers 
 md c:\drivers
-%WGET% -P c:\drivers -r -np -nH --cut-dirs=3 -R index.html -q --level=0 <%= @host.os.medium_uri(@host).to_s %>/drivers/
+%WGET% -P c:\drivers -r -np -nH --cut-dirs=<%= cut_dirs %> -R index.html -q --level=0 <%= @host.os.medium_uri(@host).to_s %>/drivers/
```

Comment:
Can you please explain in what scenario customization of this would be required?